From 9e4e4ffb107ca4f760363101999ab49225724af1 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Fri, 22 Mar 2019 08:30:28 +0900
Subject: [PATCH 761/909] arm64: dts: r8a77965: Fix power domain for FCPCS

commit a36fcaca239e26101b619b18d80d457c91786ac8 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

In R-Car Gen3 Hardware Manual Rev 1.50, the power domain of FCPCS module
was A3VC on R-Car M3N. However, this power domain is A2VC1 correctly.

NOTE: This information will be reflected in the R-Car Gen3 Hardware
      Manual in the future.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a77965.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a77965.dtsi b/arch/arm64/boot/dts/renesas/r8a77965.dtsi
index 1a4908e8ecf0..cc6dbe6a2385 100644
--- a/arch/arm64/boot/dts/renesas/r8a77965.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a77965.dtsi
@@ -2109,7 +2109,7 @@
 			compatible = "renesas,vcp4-fcpcs";
 			reg = <0 0xfe90f000 0 0x200>;
 			clocks = <&cpg CPG_MOD 619>;
-			power-domains = <&sysc R8A77965_PD_A3VC>;
+			power-domains = <&sysc R8A77965_PD_A2VC1>;
 			resets = <&cpg 619>;
 			renesas,#ch = <0>;
 		};
-- 
2.17.1

